File: test/fsharpTests/CharTest/isAlphanumericTest.fs
testCase "isAlphanumeric('A')" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Char.isAlphanumeric 'A') "error"
testCase "isAlphanumeric('?')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isAlphanumeric '?') "error"

File: test/ocamlTests/CharTest/isAlphanumericTest.ml
test "isAlphanumeric('A')" (fun () -> expect (Char.isAlphanumeric 'A') |> toEqual Eq.bool true) ; 
test "isAlphanumeric('?')" (fun () -> expect (Char.isAlphanumeric '?') |> toEqual Eq.bool false) ; 
